### PR TITLE
Reduce code size

### PR DESCRIPTION
**Pull request by ehmicky**
This PR removes some dead code.

### CODE REVIEW

File: utilities.js [hunk 1]
```diff
@@ -1,13 +1,4 @@
-export function streamToIterable(stream) {
-	return {
-		[Symbol.asyncIterator]: stream[Symbol.asyncIterator].bind(stream),
-	};
-}
```
Comment (ehmicky) [inline utilities.js:5]: Not used.

File: utilities.js [hunk 2]
```diff
@@ -1,13 +1,4 @@
...
-
-export async function * combineAsyncIterables(iterable1, iterable2) {
-	const iterator1 = iterable1[Symbol.asyncIterator]();
-	const iterator2 = iterable2[Symbol.asyncIterator]();
```
Comment (ehmicky) [inline utilities.js:9]: Node.js streams are both iterables and iterators.

Comment (sindresorhus) [reply utilities.js:9]: The function name is no longer correct then.

Comment (ehmicky) [reply utilities.js:9]: Fixed. :+1:

**Pull request merged**